src/main/java/microsoft/exchange/webservices/data/AutodiscoverService.java

```diff
@@ -327,50 +327,56 @@ private URI getRedirectUrl(String domainName) throws EWSHttpException,
     this.traceMessage(TraceFlags.AutodiscoverConfiguration, String.format(
```

Comment:
What do you mean with tracing exceptions? In this library, tracing is all about requests and responses, which are still being traced when exceptions are thrown. Do you mean the missing error logging on exceptions perhaps? I think that when the exceptions are bubbling up, we don't have to log them here.